twistr/events.go

```diff
@@ -555,6 +600,23 @@ func PlaySALTNegotiations(s *State, player Aff) {
 	   players receive -1 to all Coup Attempt rolls. The player of this card’s
 	   Event may look through the discard pile, pick any 1 non-scoring card, reveal
 	   it to their opponent and then place the drawn card into their hand. */
+	s.ImproveDefcon(2)
+	// Should this have both players involved? Or, again, is this just the player who
+	// Originally invoked the event?
+	s.TurnEvents[SALTNegotiations] = player
```

Comment:
it doesn't matter. I'm treating the player assignment here as a way of tracking who played the event. For this event specifically, we'll ignore that piece of data when checking if the event is in play.